mongodb/src/main/kotlin/com/example/entities/ArticleDto.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.entities
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ArticleDto(
```

Comment:
I would name `ArticleDto` as something referring to the action performed. For instance: `CreateArticle` if we're using it to make new article